src/c2goto/library/libm/ceil.c

```diff
@@ -25,3 +25,15 @@ ceil_def(double, ceil, rint);
 ceil_def(long double, ceill, rintl);
 
 #undef ceil_def
+
+// This function is used by the Python frontend
+void ceil_array(const double *v, double *out, int size)
```

Comment:
Can we please not add functions to headers that aren't mandated by the C standard?
Or if you absolutely want to keep it here, can we prefix them with two underscores `__` so that the name becomes a reserved name?